apps/src/templates/teacherDashboard/SectionRow.jsx

```diff
@@ -245,6 +245,15 @@ class SectionRow extends Component {
 
     const persistedSection = !!section.code;
 
+    let sectionCode = '';
+    if (!editing) {
+      if (section.providerManaged) {
+        sectionCode = '---';
+      } else {
+        sectionCode = section.code;
+      }
+    }
```

Comment:
Should we just have the server return '---' for section.code when provider managed? or do we need to use the code elsewhere?